selectors/emojis.js

```diff
@@ -39,18 +41,15 @@ export const getRecentEmojis = createSelector(
     },
 );
 
-function getStateRecentSkin(state) {
-    return state.views.emoji?.recentSkin;
-}
-
-export function getRecentSkin(state) {
-    const stateSkin = getStateRecentSkin(state);
-    if (stateSkin) {
-        return stateSkin;
-    }
-    const recentSkin = LocalStorageStore.getRecentSkin(getCurrentUserId(state));
-    return recentSkin || 'default';
-}
+// todo rename to getEmojiSkin
+export const getUserSkinTone = createSelector(
```

Comment:
We might not actually need to use createSelector here since there's nothing complicated going on in calculating this. We could just have
```javascript
export function getUserSkinTone(state) {
    return get(state, Preferences.CATEGORY_EMOJI, Preferences.EMOJI_SKINTONE, 'default');
};
```
0/5 on that though. I don't think this createSelector would actually cause any issues here.

Reply:
my original thought was so it shows up in joram's selector table, but it might not be needed as it is really simple. I'll change that.

Reply:
Ideally, I think we'd have less memoized selectors showing up there at all and more pure, simple selectors that we don't need to worry about as much :)